onnxruntime/core/providers/coreml/model/model.mm

```diff
@@ -298,23 +354,23 @@ - (Status)predict:(const std::unordered_map<std::string, OnnxTensorData>&)inputs
         return ORT_MAKE_STATUS(ONNXRUNTIME, FAIL, "output_features has no value for ", output_name);
       }
 
-      auto* data = [output_value multiArrayValue];
+      __block MLMultiArray* data = [output_value multiArrayValue];
```

Comment:
is `__block` only need for variables that we want to modify from the block?

Reply:
yep only mutables sorry about that, see other conversation :)